packages/buble/test/types.ts

```diff
@@ -1,6 +1,6 @@
 import { RollupOptions } from 'rollup';
 
-import buble from '..';
+import buble from '../types';
```

Comment:
```suggestion
import buble from '..';
```
Please switch all of these imports to use the root directory instead of the types directory.

Reply:
Oh I read the diff wrong 🤦